bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/util/CellUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+
+ * Copyright 2018 Google LLC. All Rights Reserved.
+
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.hbase.util;
+
+import java.util.Arrays;
+
+import org.apache.hadoop.hbase.Cell;
+
+/**
+ * <p> Utility methods helpful for slinging {@link Cell} instances.CellUtils class to provide utilities related to RowCell class. </p>
+ * 
+ * @author rupeshit
+ *
+ */
+public class CellUtils {
+
+  public static byte[] getValue(Cell cell) {
+    if(cell.getValueLength() >= cell.getValueArray().length) {
+      return cell.getValueArray();
+    }else if(cell.getValueLength() < cell.getValueArray().length) {
```

Comment:
Why not just "else"? 

Also, please add a space between `if` and `(`.  There should also be a space between `{` and `else`